### PR TITLE
allow us to not delete files

### PR DESCRIPTION
**Pull request by jmahowald**
It was surprising for some devs on our team to have the remote files deleted when they cleaned up locally and I figured it wouldn't hurt to give the option to turn it off.

### CODE REVIEW

File: watch [hunk 1]
```diff
@@ -61,15 +61,15 @@ function restore {
...
+  if ! $AWS s3 sync "$LOCAL" "$REMOTE" $DELETE_FLAG; then
     echo "backup failed" 1>&2
     return 1
   fi
 }
 
 function final_backup {
   echo "backup $LOCAL => $REMOTE"
-  while ! $AWS s3 sync "$LOCAL" "$REMOTE" --delete; do
+  while ! $AWS s3 sync "$LOCAL" "$REMOTE" $DELETE_FLAG; do
```
Comment (fabiob) [inline watch:72]: I think `S3_SYNC_FLAGS` is a better name.

**Review by fabiob — CHANGES_REQUESTED** [review]
I like the approach of exposing the sync flags so the user can change them. If you can change the variable name to `S3_SYNC_FLAGS`, I'll merge it.

File: watch (hunk 1 above)
Comment (mirsal) [inline watch:72]:
```suggestion
  while ! $AWS s3 sync "$LOCAL" "$REMOTE" $S3_SYNC_FLAGS; do
```

File: watch [hunk 2]
```diff
@@ -61,15 +61,15 @@ function restore {
 
 function backup {
   echo "backup $LOCAL => $REMOTE"
-  if ! $AWS s3 sync "$LOCAL" "$REMOTE" --delete; then
+  if ! $AWS s3 sync "$LOCAL" "$REMOTE" $DELETE_FLAG; then
```
Comment (mirsal) [inline watch:64]:
```suggestion
  if ! $AWS s3 sync "$LOCAL" "$REMOTE" $S3_SYNC_FLAGS; then
```

File: README.md [hunk 3]
```diff
@@ -80,6 +80,9 @@ docker run -d --name my-data-container \
            elementar/s3-volume --force-restore /data s3://mybucket/someprefix
 ```
 
+### Deletion and sync
+By default if there are files that are deleted in your local file system, those will be deleted remotely.   If you wish to turn this off set the environment variable DELETE_FLAGS=":
```
Comment (mirsal) [inline README.md:84]:
```suggestion
By default if there are files that are deleted in your local file system, those will be deleted remotely.   If you wish to turn this off set the environment variable S3_SYNC_FLAGS=":
```

File: Dockerfile [hunk 4]
```diff
@@ -6,5 +6,6 @@ ADD watch /watch
 
 VOLUME /data
 
+ENV DELETE_FLAG "--delete"
```
Comment (mirsal) [inline Dockerfile:9]:
```suggestion
ENV S3_SYNC_FLAGS "--delete"
```

**Review by fabiob — APPROVED** [review]

**Pull request merged**